test/unit/plugins/provisioners/chef/config/base_test.rb

```diff
@@ -82,4 +82,11 @@
       expect(subject.installer_download_path).to be(nil)
     end
   end
+
+  describe "#omnibus_url" do
+    it "defaults to https://omnitruck.chef.io" do
+      subject.finalize!
+      expect(subject.omnibus_url).to eq("https://omnitruck.chef.io")
+    end
```

Comment:
Can we also get a test that asserts if the user sets it to something other than the default, it gets properly used?